frameworks/helloworld/tests/test_secrets.py

```diff
@@ -72,7 +72,7 @@ def teardown_module(module):
 @pytest.mark.sanity
 @pytest.mark.smoke
 @pytest.mark.secrets
-@dcos_1_10
+@pytest.mark.skipif('shakedown.dcos_version_less_than("1.10")')
```

Comment:
Cool. Can you use the annotation I created in sdk_utils for this? You just need to import shakedown (already done) and sdk_utils and it will work.

https://github.com/mesosphere/dcos-commons/blob/master/testing/sdk_utils.py#L46